src/components/HistoryTable/HistoryTable.tsx

```diff
@@ -12,21 +13,35 @@ interface HistoryTableProps {
   loadMoreItems: () => void;
   recentlyFetchedCommits: MainlineCommitsForHistoryQuery["mainlineCommits"];
   children: ComponentType<ListChildComponentProps<any>>;
+  height: number;
+  width: number;
 }
-const HistoryTable: React.FC<HistoryTableProps> = ({
+const HistoryTable = ({
   loadMoreItems,
   recentlyFetchedCommits,
   children,
-}) => {
+  height,
+  width,
+}: HistoryTableProps): JSX.Element => {
   const {
     getItemHeight,
     fetchNewCommit,
     isItemLoaded,
     toggleRowSizeAtIndex,
     commitCount,
+    onChangeTableWidth,
   } = useHistoryTable();
   const listRef = useRef<List>(null);
 
+  const throttleOnChangeTableWidth = useMemo(
+    () => throttle(onChangeTableWidth, 400),
+    [onChangeTableWidth]
+  );
+
+  useEffect(() => {
+    throttleOnChangeTableWidth(width);
+  }, [width, throttleOnChangeTableWidth]);
```

Comment:
https://github.com/bvaughn/react-virtualized/blob/master/docs/AutoSizer.md Looks like autosizer supports an onResize callback. We don't need this useEffect.